src/pages/home/report/ReportFooter.js

```diff
@@ -122,4 +119,19 @@ export default compose(
             initialValue: false,
         },
     }),
-)(ReportFooter);
+)(
```

Comment:
is it also required even with the lightweight report object?

Reply:
Yes the reason is that, the lightweight report object is created in `ReportActionsList` and not in `ReportScreen`. The reason is that in `ReportScreen` we are passing `report` object to a handful of utility functions and having the lightweight object here is rather complicated.